helsingborg-stad/aterbruket-webapp
# Feature/separate items haffat

**Explain the changes you’ve made**
Separated the reserved items and the picked up items
Put a button "hämta ut" on the card to make it more user friendly

**Explain why these changes are made**
So it looks like the design. And easier for the user to hämta without going in to the item detail page

**Explain your solution**
I did the function for updateItem inside the card-component, and then the get function to render out the cards right

**How to test the changes?**
- Haffa something
- Go to haffat-page and see that the items are separated depending on its status
- See the hämta-ut button on the reserved items, click it and hope it works
